cas-server-core-api-ticket/src/main/java/org/jasig/cas/ticket/registry/TicketRegistryState.java

```diff
@@ -15,7 +15,7 @@
      * @return Number of ticket-granting tickets in the registry at time of invocation
      *         or {@link Integer#MIN_VALUE} if unknown.
      */
-    int sessionCount();
+    long sessionCount();
```

Comment:
Most sessionCount() impls do this:

```
 getTickets().stream().filter(t -> t instanceof TicketGrantingTicket).count();
```

that returns a long. 

Also, theoretically it's possible for the session count to be a long if you have a billion sessions available and thousands and thousands of transactions going through. As such, the API here would simply bomb with an int.